Update documentation for spill

rschlussel opened the pull request:
Test plan - build and look at docs

```
== NO RELEASE NOTE ==
```

mzbeck reviewed (changes requested):
I don't see definitions for some of the properties that you mention in your diff. 2 examples:  order_by_aggregation_spill_enabled and  distinct_aggregation_spill_enabled session property. Shouldn't they also be documented in properties.rst?

mzbeck commented on presto-docs/src/main/sphinx/admin/properties.rst:
```diff
@@ -151,6 +151,96 @@ Spilling Properties
+``experimental.memory-revoking-threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    * **Type:** ``double``
+    * **Default value:** ``0.9``
```
It's not clear to me what 0.9 refers to ie. the unit. is it a percentage / MB etc

rschlussel replied:
clarified that it 's percentage (not sure if that's the right word since it's a value between 0 and 1 rather than 1 and 100)

mzbeck commented on presto-docs/src/main/sphinx/admin/properties.rst:
```diff
@@ -151,6 +151,96 @@ Spilling Properties
+``experimental.memory-revoking-target``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    * **Type:** ``double``
+    * **Default value:** ``0.5``
```
It's not clear to me what 0.5 refers to ie. the unit. is it a percentage / MB etc

rschlussel replied:
clarified that it's percentage

rschlussel commented:
Thanks @mzbeck for the review!  I've addressed your comments.

> I don't see definitions for some of the properties that you mention in your diff. 2 examples: order_by_aggregation_spill_enabled and distinct_aggregation_spill_enabled session property. Shouldn't they also be documented in properties.rst?

Based on the rest of the doc, I think we just document config properties and note the equivalent session properties where relevant rather than having separate documentation for the session properties.
